### DISCUSSION TITLE
Remassage the output logs to s3 for AWS glue

### DISCUSSION THREAD

**Discussion by jfdoube** [cloud-custodian/cloud-custodian#7425] [AWS]
### Ask your question

AWS glue -> AWS Athena is very complex due to the format of the logs, especially the json one since it contain much more information. Any proposed existing solution or some feature that could be implements ?

Thank you

**Comment by herrsergio** [comment]
It's related with this issue: https://github.com/cloud-custodian/cloud-custodian/issues/7051
